.goreleaser.yml

```diff
@@ -21,7 +21,7 @@ dockers:
     - "inngest/inngest:{{ .Tag }}-amd64"
     build_flag_templates:
     - "--pull"
-    - "--build-arg=BIN=./build/inngest-linux-amd64"
```

Comment:
Checking the logs of the last release, `xgo` seems to run this command: which outputs into the `./build` dir:
```
go build -trimpath --mod=vendor '--ldflags= -s -w -X github.com/inngest/inngest/pkg/inngest/version.Version=v0.15.3 -X github.com/inngest/inngest/pkg/inngest/version.Hash=c75c0d51' -o /build/inngest-linux-amd64 ./cmd
```

https://github.com/inngest/inngest/blob/bee5ef1b1c1fd987cf7f981cddd902fa37642b02/.github/workflows/release.yml#L47

And the goreleaser seems to correctly copy it here: https://github.com/inngest/inngest/actions/runs/5557582337/jobs/10151552533#step:13:159 

Is there some context that I'm missing about the build or Github workflow? I'd love to more fully understand this change and how it works :)

Reply:
Goreleaser copies from the build output, eg. the build folder, so we dont need to use relative paths.